webpack.config.js

```diff
@@ -5,7 +5,7 @@ var webpack = require('webpack');
 
 var base = {
     devServer: {
-        contentBase: path.resolve(__dirname, 'playground'),
+        contentBase: false,
```

Comment:
If I'm interpreting [the docs](https://webpack.js.org/configuration/dev-server/#devserver-contentbase) correctly the default is equivalent to `contentBase: __dirname`, which would mean that (for example) `http://localhost:8073/package.json` would be available.

Setting `contentBase` to `false` turns off static content serving altogether... I feel like that _should_ be the default, but it isn't.